tutorials/btp-hyperscaler-extension/btp-hyperscaler-extension.md

```diff
@@ -8,11 +8,11 @@ primary_tag: products>sap-btp--cloud-foundry-environment
 ---
 
 ## Prerequisites
- - You have either an [enterprise](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/171511cc425c4e079d0684936486eee6.html) or a [trial](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/046f127f2a614438b616ccfc575fdb16.html) global account in SAP BTP
+- Get an Account on SAP BTP to Try Out Free Tier Service Plans.follow steps 1 to 8 from this tutorial [Get an Account on SAP BTP](btp-free-tier-account)
```

Comment:
A prerequisite is not an instruction! Please rephrase!

Reply:
I have made the changes please have a look